client/my-sites/plan-features-list/index.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PlanFeatures from 'my-sites/plan-features';
+import { PLAN_FREE, PLAN_PREMIUM, PLAN_BUSINESS } from 'lib/plans/constants';
+import FAQ from 'components/faq';
+import FAQItem from 'components/faq/faq-item';
+
+class PlanFeaturesList extends Component {
+
+	render() {
+		const { translate, site } = this.props;
+
+		return (
+			<div>
+				<div className="plan-features-list">
```

Comment:
Agreed. Is `PlansFeaturesPage` good enough? The page contains not only plans but also FAQs, that's why I'm thinking of calling it a 'page'.